apps/mobile-alphabet/src/app/Components/alphabet-card-detail.screen.tsx

```diff
@@ -48,8 +61,7 @@ export function AlphabetCardDetailScreen() {
         standalone_image,
     } = selectedCard;
 
-    //TODO pull this from a config
-    const apiUrlPrefix = `http://10.0.2.2:3131/api/resources/mediaitems/download?name=`;
+    const apiUrlPrefix = config.apiUrlPrefix;
```

Comment:
Can you use a `ConfigContext` to inject this state instead? See coscrad-frontend.